scripts/runmariadb

```diff
@@ -16,14 +16,12 @@ if [ ! -f "$MARIADB_CERT_FILE" ] && [ -f "$MARIADB_KEY_FILE" ] ; then
     exit 1
 fi
 
-ln -sf /proc/self/fd/1 /var/log/mariadb/mariadb.log
-rm -f /usr/bin/mysqld_safe_helper # Avoid this script being used which results in a permission denied error.
-
 if [ ! -d "${DATADIR}/mysql" ]; then
     crudini --set "$MARIADB_CONF_FILE" mysqld max_connections 64
     crudini --set "$MARIADB_CONF_FILE" mysqld max_heap_table_size 1M
     crudini --set "$MARIADB_CONF_FILE" mysqld innodb_buffer_pool_size 5M
     crudini --set "$MARIADB_CONF_FILE" mysqld innodb_log_buffer_size 512K
+    crudini --set "$MARIADB_CONF_FILE" mariadb-10.3 skip_log_error # Error log will be redirected to stderr
```

Comment:
Are you sure the group name is actually "mariadb-10.3", not mysqld as above?

Reply:
It should work in both group, I guess. I choose "mariadb-10.3" since in the example they use a similar group. https://mariadb.com/kb/en/error-log/#writing-the-error-log-to-stderr-on-unix.